Tasks/AndroidSigningV3/task.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "80F3F6A0-82A6-4A22-BA7A-E5B8C541B9B9",
+    "name": "AndroidSigning",
+    "friendlyName": "Android Signing",
+    "description": "Sign and align Android APK files",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613717)",
+    "category": "Build",
+    "visibility": [
+        "Build"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 3,
+        "Minor": 137,
+        "Patch": 0
+    },
+    "demands": [
+        "JDK",
+        "AndroidSDK"
+    ],
```

Comment:
Add "releasenotes" to explain what changed in this version. https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/XcodeV5/task.json#L17